scripts/jlab-env.csh.in

```diff
@@ -1,3 +1,4 @@
 module load cmake/@CMAKE_VERSION@
 module load gcc/@CMAKE_C_COMPILER_VERSION@
 module load python/@Python3_VERSION@
+source /work/hallb/hps/bravo/sw/root/buildV62202Py3/bin/thisroot.csh
```

Comment:
I don't like this because it now configures any job that runs at JLAB to use a specific ROOT version, and the hps-mc being used may not work with it. Can't this be done with CMake options? It should be generic even if the script is site-specific. (I don't remember if the hps-mc CMake configures ROOT to know where ROOTSYS is located but that would be easy to add if not.)